components/frontend/src/widgets/DivWithHTML.jsx

```diff
@@ -1,3 +1,5 @@
+import "./DivWithHTML.css"
```

Comment:
Huh, odd import that isn't referenced by name exactly. Is that because it's a css import, doesn't a top level css import suffice?

Reply:
This is indeed how importing CSS-files works in React applications.